ova/Libraries/provision-opendistro.sh

```diff
@@ -236,7 +236,7 @@ installKibana() {
         fi
         mkdir -p /etc/kibana/certs
         cp /etc/elasticsearch/certs/root-ca.pem /etc/kibana/certs/
-        mv /etc/elasticsearch/certs/{kibana.key,kibana.pem} /etc/kibana/certs/
+        mv /etc/elasticsearch/certs/{kibana_http.key,kibana_http.pem} /etc/kibana/certs/
```

Comment:
The `kibana.yml` file that is downloaded in this line:

https://github.com/wazuh/wazuh-packages/blob/c518fb2d2184516e17dbd4bc410524aab39d1efa/ova/Libraries/provision-opendistro.sh#L218

References the certificate files as `kibana.key` and `kibana.pem`

https://github.com/wazuh/wazuh-documentation/blob/0dbb994f2c9c8f60d7d1b36330d65ed349d4c5c4/resources/open-distro/kibana/7.x/kibana_all_in_one.yml#L11

We need to rename them or change the `kibana.yml` file.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/534/commits/6166e85603757c16f5ae469be9373dcdecf6751a